packages/form-js-playground/src/Playground.js

```diff
@@ -117,10 +117,22 @@ export default function Playground(options) {
     return ref.get(name, strict);
   });
 
+  this.getDataEditor = withRef(function() {
+    return ref.getDataEditor();
+  });
```

Comment:
`ref` and its methods are lazy loaded and not available on the first instantiation. Although I'd like the shorter format, we need to make sure `ref` is accessed only when actual using the API method ([after the playground is initiated properly](https://github.com/bpmn-io/form-js/blob/e6b605dfab63a0964ae15da17e1c105ce5748df0/packages/form-js-playground/src/Playground.js#L68)).

Reply:
Took some time to wrap my head around this one, but yeah this makes a lot of sense, even if the withRef pattern feels somewhat obtuse to me. 

Little JSFiddle where I'm experimenting with it:
https://jsfiddle.net/shbw0gmo/5/

Anyways, I'd suggest we just use arrow functions here instead for simplicity and readability. It conveys the idea that we're forwarding a function to another quite well visually.

Reply:
For reference, we use something similar in the Desktop Modeler: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/App.js#L2310

Fail safe with giving enough context, instead of randomly fail.

Reply:
I agree that arrow functions improve the situation :+1:

Reply:
Introducing arrow functions via https://github.com/bpmn-io/form-js/pull/364/commits/b3d8457089cf8d6cb72e43829a508f500cacd5b1.